### PR TITLE
adding faq/bug fix to bottom of delegation docs

### PR DESCRIPTION
**Pull request by brandonleekramer**
Brandon (UX/Design) found this bug a couple weeks ago while attempting to delegate and while scanning Discord for other data. Brandon shared this with the Product Engineering team (Nas) in Asana and the Graphtronauts on Telegram before Adam Fuller suggested that this FAQ/Bug would be best shared on the docs page.

### CODE REVIEW

File: pages/en/delegating.mdx [hunk 1]
```diff
@@ -83,6 +83,14 @@ Imagine an Indexer has 100,000,000 GRT delegated to them, and their capacity is
...
+
+At times, attempts to delegate to indexers via MetaMask can fail and result in prolonged periods of "Pending" or "Queued" transaction attempts. For example, a user may attempt to delegate and run out of ETH to cover the gas costs to complete the transaction, resulting in the transaction attempt displaying as "Pending" in their MetaMask wallet for 15+ minutes. When this occurs, subsequent transactions can be attempted by a user, which pushes the initial transaction into a queue and raises the most recent attempt(s) to "Pending" in place of the initial attempt. In such cases, these transactions can be cancelled in MetaMask, but the transactions attempts will accrue gas fees without any guarantee that subsequent attempts will be successful. The cheapest known fix to this bug is restarting the browsesr (e.g., using "abort:restart" in the address bar), which will cancel all previous attempts without gas being subtracted from the wallet. Several users that have encountered this issue and have reported successful transactions after restarting their browser and attempting to delegate.
```
Comment (azf20) [inline pages/en/delegating.mdx:92]: Some edits for accuracy:

> delegate and run out of ETH to cover the gas costs to complete the transaction

...delegate with an insufficient gas fee, given current prices...

>  which pushes the initial transaction into a queue and raises the most recent attempt(s) to "Pending" in place of the initial attempt

... but these will not be processed until the initial transaction is mined, as transactions for an address must be processed in order.

> the transactions attempts will accrue gas fees without any guarantee that subsequent attempts will be successful

... the cancellation transactions will incur gas fees.

> The cheapest known fix to this bug

A simpler resolution

**Review by azf20 — CHANGES_REQUESTED** [review]
Hey thanks! Suggested some edits to the explanation.

**Comment by brandonleekramer** [comment]
Thank you, Adam. I've implemented these suggestions into the docs and pushed again.

File: pages/en/delegating.mdx [hunk 2]
```diff
@@ -83,6 +83,14 @@ Imagine an Indexer has 100,000,000 GRT delegated to them, and their capacity is
...
+
+At times, attempts to delegate to indexers via MetaMask can fail and result in prolonged periods of "Pending" or "Queued" transaction attempts. For example, a user may attempt to delegate with an insufficient gas fee relative to the current prices, resulting in the transaction attempt displaying as "Pending" in their MetaMask wallet for 15+ minutes. When this occurs, subsequent transactions can be attempted by a user, but these will not be processed until the initial transaction is mined, as transactions for an address must be processed in order. In such cases, these transactions can be cancelled in MetaMask, but the transactions attempts will accrue gas fees without any guarantee that subsequent attempts will be successful. A simpler resolution to this bug is restarting the browsesr (e.g., using "abort:restart" in the address bar), which will cancel all previous attempts without gas being subtracted from the wallet. Several users that have encoA simpler resolutionuntered this issue and have reported successful transactions after restarting their browser and attempting to delegate.
```
Comment (azf20) [inline pages/en/delegating.mdx:92]: sorry last one on this - seems to be a typo
```
encoA simpler resolutionuntered
```

**Review by azf20 — CHANGES_REQUESTED** [review]

**Review by azf20 — APPROVED** [review]

**Comment by azf20** [comment]
LGTM, thanks!

**Pull request merged**